proposed/2021/NuGetOrgDetailsPageRedesign.md

```diff
@@ -0,0 +1,140 @@
+
+### Why: How do we know this is a real problem and worth solving?
+
+We have received many requests for UI improvements on GitHub. Developers have mentioned consistently in customer interviews that they do not like to use NuGet.org for finding and validating  packages and will look for other resources whenever they can. The information is not organized intuitively and caused users to scroll to the bottom of the page or use extra clicks. In addition, about 20% of our traffic comes from mobile usage and we need to make sure our design continues to work with mobile.   
+
+We cannot improve the NuGet.org package details page entirely with small, undirected changes. Instead we need a coherent, comprehensive, and most importantly, modular plan of action that addresses the core problems but can be easily separated into manageable chunks of work.
+
+
+### Success: How do we know if we’ve solved this problem?
+- By HaTS survey results and customer interview replies  
+- By measuring rates at which people click on sections/tabs on the package details page  
+- By measuring changes in the NuGet.org traffic for the details page of our top 100 popular packages
+- By gauging excitement/disappointment after blogging about this feature & analyzing sentiment on Twitter/GitHub/DevCom/etc.
+
+
+### Audience: Who are we building for?
+
+We are building this for the .NET developer who is browsing & making decisions about including a NuGet package. They want to:
+-	Find information they need quickly and easily
+-	Use higher quality packages
+-	Use more secure packages
+
+We are also building this for the .NET developer who is creating & publishing NuGet packages for public use. They want to:
+-	Display relevant information about their package
+-	Increase usage of their package
+
+## Goals and Non-Goals
+
+### Header
+
+![](../../meta/resources/NuGetPackageDetails/headernew.png)
+
+Information that needs to be seen at a glance such as package ID, description, and eventually, TFM badges.
+
+**Requirements**        
+- Title   
+- Icon  
+    - Hidden for packages without icon
+- Version   
+- Checkmark to indicate reserved prefix     
+  - includes link to prefix reservation docs
+- Installation instructions
+  - Dropdown for choosing variant
+  - Sticks to chosen variant for all future browsing
+  - Additional, detailed installation instructions for new users in Installation Tab
+- Vulnerability and other critical warning banners 
+  - will be shown at very top of page 
+  - catches immediate attention
+
+**Non-Requirements**
+- Target Framework badges
+  - It's coming, just in a separate proposal!
+
+
+### Sidebar
+
+Additional helpful information such as package installation instructions, statistics, external links. 
+
+![](meta/resources/NuGetPackageDetails/sidebar_final.png)
+
+**Requirements:**
+- Download statistics 
+  - Total downloads
+  - Downloads of current version
+  - Average downloads per day
```

Comment:
Are downloads per day of the current version or the overall package? We may want to consider only showing 2 of these to save on space and make it more focused. 

My vote would be keep Total and Current Version downloads and put the Per Day in the "full stats" page. It may be helpful to get customer feedback on that tho 😊

Reply:
Overall package!

That makes sense and I agree that we should be more intentional about what stats we show vs. hide on the "full stats" page. For this work, however, I plan on keeping these three just because they are the three displayed on the current package details page. Good to have a baseline to compare against as we start modifying how/what we show instead of just the way we show information.

Reply:
Can we keep a running list of considered small design fixes that we'd like to test out after the initial iteration? My concern being that most of these are minor issues we may struggle to individually prioritize once the bulk of this feature is complete. It'll probably we overall less work to review and address them as we're refining this redesign 😊

Your approach for the initial iteration makes complete sense to me 👍